validator_test.go

```diff
@@ -231,6 +231,8 @@ func parseChange(str string) *change {
 		}
 		st.Mode |= uint32(os.ModeSymlink)
 		st.Linkname = f[3]
+	default:
+		panic(fmt.Sprint("unrecognized file type:", typ))
```

Comment:
Added this to find suspected typos in input (see above). If this is intended - please let me know.